book/src/basics/grafana.md

```diff
@@ -0,0 +1,13 @@
+# Grafana
+
+After `solana-exporter` exports gauges and metrics to Prometheus, it may be useful to visualise these metrics.
+[Grafana](https://grafana.com/) allows you to create custom dashboards using Prometheus as a data source.
```

Comment:
Can you specify how to add the Prometheus data source and how to import the dashboard and use that data source in the dashboard? Screenshots of the Grafana admin panel can help a lot here.

Reply:
I think we should link to official Grafana documentation on:
- How to add a Prometheus data source
- How to import an existing dashboard

honestly. Official docs can be guaranteed to be always correct and up to date for any changes to Grafana in the future.

Reply:
Fixed in 7307f5c?